Language/Functions/USB/Keyboard/keyboardModifiers.adoc

```diff
@@ -14,57 +14,82 @@ title: Keyboard Modifiers and Special Keys
 
 [float]
 === Description
-The `Keyboard.write()` and `Keyboard.press()` and `Keyboard.release()` commands don’t work with every possible ASCII character, only those that correspond to a key on the keyboard. For example, backspace works, but many of the other non-printable characters produce unpredictable results. For capital letters (and other keys), what’s sent is shift plus the character (i.e. the equivalent of pressing both of those keys on the keyboard).
+When given a printable ASCII character as an argument, the functions `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()` simulate actuations on the corresponding keys. These functions can also handle ASCII characters that require pressing a key in combination with Shift or, on international keyboards, AltGr. For example:
+[source,arduino]
+----
+Keyboard.write('a');  // press and release the 'A' key
+Keyboard.write('A');  // press Shift and 'A', then release both
+----
+A typical keyboard, however, has many keys that do not match a printable ASCII character. In order to simulate those keys, the library provides a set of macros that can be passed as arguments to `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()`. For example, the key combination Shift-F2 can be generated by:
+[source,arduino]
+----
+Keyboard.press(KEY_LEFT_SHIFT);  // press and hold Shift
+Keyboard.press(KEY_F2);          // press and hold F2
+Keyboard.releaseAll();           // release both
+----
+Note that, in order to press multiple keys simultaneously, one has to use link:../keyboardpress[`Keyboard.press()`] rather than link:../keyboardwrite[`Keyboard.write()`], as the latter just “hits” the keys (it presses and immediate releases them).
 [%hardbreaks]
-For more on ASCII values and the characters or functions they represent, see http://www.asciitable.com/[asciitable.com]
+The available macros are listed below:
 
 [float]
 === Keyboard modifiers
-A modifier key is a special key on a computer keyboard that modifies the normal action of another key when the two are pressed in combination.
-[%hardbreaks]
-For multiple key presses use link:../keyboardpress[Keyboard.press]()
-[%hardbreaks]
-The definitions of the modifier keys are listed below:
-[%hardbreaks]
-
+These are eight keys within the alphanumeric cluster of the keyboard that are meant to modify the normal action of another key when the two are pressed in combination. Whereas Shift and AltGr are used to access extra characters, the others are mostly used for keyboard shortcuts.
```

Comment:
```suggestion
These keys are meant to modify the normal action of another key when the two are pressed in combination.
```
Reduce unnecessary verbosity. We can assume the reader is familiar with the use of a keyboard.